project-keeper/src/main/java/com/exasol/projectkeeper/validators/files/FileTemplatesFactory.java

```diff
@@ -19,12 +19,27 @@ class FileTemplatesFactory {
     private static final String POM_FILES_GENERATED = String.format("%-65s%s", "pk_generated_parent.pom",
             "linguist-generated=true");
 
+    private static final String NPM_SETUP = lines( //
+            "      - name: Set up NPM", //
+            "        uses: actions/setup-node@v3", //
+            "        with:", //
+            "          node-version: 16", //
```

Comment:
```suggestion
            "          node-version: 18", //
```
18 is the latest LTS version